src/DssSpell.t.sol

```diff
@@ -528,81 +508,86 @@ contract DssSpellTest is DssSpellTestBase {
-
-        // 20220907 exec: Warp 1/2 year since cliff, ensure vest was accumulated
-        hevm.warp(JUL_01_2022 + 365 days / 2);
-        vest.vest(24);
-        assertEq(gov.balanceOf(GRO_WALLET), prevBalance + (803 * WAD / 2));
-
-        // 20220907 exec: Warp to end and vest remaining
-        hevm.warp(JUL_01_2022 + 365 days);
-        vest.vest(24);
-        assertEq(gov.balanceOf(GRO_WALLET), prevBalance + 803 * WAD);
-
-        // 20220907 exec: Warp even further and make sure nothing else was accumulated
-        hevm.warp(JUL_01_2022 + 365 days + 10 days);
-        vest.vest(24);
-        assertEq(gov.balanceOf(GRO_WALLET), prevBalance + 803 * WAD);
+        assertEq(vest.usr(25), GOV_WALLET1);
+        assertEq(vest.bgn(25), AUG_01_2022);
+        assertEq(vest.clf(25), AUG_01_2023);
+        assertEq(vest.fin(25), AUG_01_2023);
+        assertEq(vest.mgr(25), address(0));
+        assertEq(vest.res(25), 1);
+        assertEq(vest.tot(25), 62.50 ether);
+        assertEq(vest.rxd(25), 0);
+
+        assertEq(vest.usr(26), GOV_WALLET2);
+        assertEq(vest.bgn(26), AUG_01_2022);
+        assertEq(vest.clf(26), AUG_01_2023);
+        assertEq(vest.fin(26), AUG_01_2023);
+        assertEq(vest.mgr(26), address(0));
+        assertEq(vest.res(26), 1);
+        assertEq(vest.tot(26), 32.69 ether);
+        assertEq(vest.rxd(26), 0);
+
+        assertEq(vest.usr(27), GOV_WALLET3);
+        assertEq(vest.bgn(27), AUG_01_2022);
+        assertEq(vest.clf(27), AUG_01_2023);
+        assertEq(vest.fin(27), AUG_01_2023);
+        assertEq(vest.mgr(27), address(0));
+        assertEq(vest.res(27), 1);
+        assertEq(vest.tot(27), 152.51 ether);
+        assertEq(vest.rxd(27), 0);
+
+        assertEq(vest.usr(28), SNE_WALLET);
+        assertEq(vest.bgn(28), SEP_28_2022);
+        assertEq(vest.clf(28), SEP_28_2022);
+        assertEq(vest.fin(28), SEP_28_2024);
+        assertEq(vest.mgr(28), address(0));
+        assertEq(vest.res(28), 1);
+        assertEq(vest.tot(28), 540.00 ether);
+        assertEq(vest.rxd(28), 0);
     }
 
-    function testMKRPayments() public { // make private to disable
+    function testMKRPayments() public {
         uint256 prevMkrPause = gov.balanceOf(address(pauseProxy));
-        uint256 prevMkrRWF   = gov.balanceOf(wallets.addr("RWF_WALLET"));
-        uint256 prevMkrCES   = gov.balanceOf(wallets.addr("CES_OP_WALLET"));
-        uint256 prevMkrRisk  = gov.balanceOf(wallets.addr("RISK_WALLET_VEST"));
+        uint256 prevMkrSNE   = gov.balanceOf(wallets.addr("SNE_WALLET"));
+        uint256 prevMkrSES   = gov.balanceOf(wallets.addr("SES_WALLET"));
 
-        uint256 amountRWF    =  20.00 ether;
-        uint256 amountCES    = 966.49 ether;
-        uint256 amountRisk   = 175.00 ether;
+        uint256 amountSNE    = 270.00 ether;
+        uint256 amountSES    = 227.64 ether;
 
-        uint256 total = amountRWF + amountCES + amountRisk;
+        uint256 total = amountSNE + amountSES;
```

Comment:
we could also use the explicit `total` to ensure it's correct and matches the exec copy, if stated.